### ISSUE TITLE
Feature request: From Chrome, goto line in Sublime

### ISSUE THREAD

**Issue by eablokker** [mechio/takana#85]
I would really love to be able to click on an element or a selector in Chrome and have Takana take me to the line where that CSS is declared in Sublime. Seems like it should be possible, to get the line number of a selector in Chrome, and send it back over the Takana server back to Sublime.

**Comment by ob7** [comment]
I'm not sure if that would be a plugin specifically for Takana.  It seems that would be a plugin all by itself.

**Comment by eablokker** [comment]
Yeah, could be a separate plugin. Just thought I'd put that idea out there.
